src/pages/dashboard/DashboardPage.tsx

```diff
@@ -115,30 +132,42 @@ function DashboardPage() {
       resizeHandle={renderResizeHandle()}
       containerPadding={[0, 0]}
     >
-      <div key="queueProcessor">
-        <QueueProcessor />
-      </div>
+      {!hideQueueProcessor && (
+        <div key="queueProcessor">
+          <QueueProcessor />
+        </div>
+      )}
       <div key="importBreakdown">
         <UnrecognizedFiles />
```

Comment:
Why can't we hide the unrecognized files? @ElementalCrisis @harshithmohan

Reply:
I added the option to hide it.